pkg/slayers/path/scion/raw_spec.gobra

```diff
@@ -850,38 +844,52 @@ func XoverRightSeg(raw []byte, currInfIdx int, currHfIdx int, segs io.SegLens) {
 }
 
 ghost
-opaque
+requires  R55 < p
```

Comment:
The usage of an additional parameter here is unfortunate for the clients. This is an implementation detail and the clients shouldn't have to provide this parameter. I would make this an auxiliar function that is called from a public lemma that is meant to be called without having to pass this parameter. The same applies to all lemmas that take this parameter.